.github/workflows/check_nims.yml

```diff
@@ -12,6 +12,10 @@ jobs:
   check_nims:
     name: Check NIMS
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        # Set the working-directory for all steps in this job.
+        working-directory: ./packages/service
```

Comment:
I don't think this affects actions/cache because it is not a "run" step. It is probably caching an empty/nonexistent .venv directory now. This goes for all of the caching in the workflows.

Reply:
Yeah, it's also computing a bad hash key because `${{ hashFiles('poetry.lock') }}` evaluates to an empty string when it can't find `poetry.lock`.

https://github.com/ni/measurement-plugin-python/actions/runs/9863829285/job/27237445507?pr=781
> Cache restored from key: ni-measurement-plugin-sdk-service-all-extras-Linux-py3.9.[19](https://github.com/ni/measurement-plugin-python/actions/runs/9863829285/job/27237445507?pr=781#step:6:20)-

Reply:
So we change it to `hashFiles('packages/service/poetry.lock')`? I'll try that. Or is there some other 'working-directory' type property on the cache action?